conf/messages

```diff
@@ -297,7 +297,7 @@ chargeType.miscPenaltyCharge = VAT general penalty
 chargeType.VatOfficersAssessmentFurtherInterestTitle = VAT officer’s assessment further interest
 chargeType.VatOfficersAssessmentFurtherInterestDescription = charged on the officer’s assessment
 chargeType.VatAdditionalAssessmentTitle = Additional assessment
```

Comment:
According to the spreadsheet it would be "Additional assessment of VAT" for chargeType.VatAdditionalAssessmentTitle